packages/cli/package.json

```diff
@@ -74,14 +75,16 @@
     "plurals-cldr": "^1.0.4",
     "pofile": "^1.1.4",
     "pseudolocale": "^1.1.0",
-    "ramda": "^0.27.1"
+    "ramda": "^0.27.1",
+    "source-map": "^0.8.0-beta.0"
```

Comment:
use beta because stable has known issue where environment not detected correctly and it fails to load wasm module.